test/test_autogluon_tabular.py

```diff
@@ -0,0 +1,80 @@
+import pandas as pd
+import numpy as np
+from gluonts.nursery.autogluon_tabular import LocalTabularPredictor
+from gluonts.dataset.common import ListDataset
+from gluonts.evaluation import Evaluator
+
+
+def test_autogluon_tabular(verbose=False):
+    # create a dataset
+    dataset = ListDataset(
+        [
+            {
+                "start": pd.Timestamp("1750-01-04 00:00:00", freq="W-SUN"),
+                "target": np.array(
+                    [1089.2, 1078.91, 1099.88, 35790.55, 34096.95, 34906.95],
+                ),
+            },
+            {
+                "start": pd.Timestamp("1750-01-04 00:00:00", freq="W-SUN"),
+                "target": np.array(
+                    [1099.2, 1098.91, 1069.88, 35990.55, 34076.95, 34766.95],
+                ),
+            },
+            {
+                "start": pd.Timestamp("1750-01-04 00:00:00", freq="W-SUN"),
+                "target": np.array(
+                    [1079.2, 1078.91, 1029.88, 35790.55, 34056.95, 34566.95],
+                ),
+            },
+            {
+                "start": pd.Timestamp("1750-01-04 00:00:00", freq="W-SUN"),
+                "target": np.array(
+                    [2059.2, 1057.91, 1019.88, 35590.55, 34036.95, 34366.95],
+                ),
+            },
+            {
+                "start": pd.Timestamp("1750-01-04 00:00:00", freq="W-SUN"),
+                "target": np.array(
+                    [1089.2, 1078.91, 1099.88, 35790.55, 34056.95, 34566.95],
+                ),
+            },
+            {
+                "start": pd.Timestamp("1750-01-04 00:00:00", freq="W-SUN"),
+                "target": np.array(
+                    [1049.2, 1057.91, 1049.88, 35590.55, 34036.95, 34366.95],
+                ),
+            },
+        ],
+        freq="W-SUN",
+    )
+
+    predictor = LocalTabularPredictor(freq="W-SUN", prediction_length=1,)
```

Comment:
You can try with different `prediction_length` settings here, by looping through different values